_templates/layout.html

```diff
@@ -45,20 +45,19 @@
 
     {% if godot_show_article_status %}
     <div class="admonition tip article-status">
+      {% if meta and meta.get('article_outdated') == 'True' %}
       <p class="first admonition-title">Work in progress</p>
       <p>
-        Godot documentation is being updated to reflect the latest changes in version
-        <code class="docutils literal notranslate">{{ godot_version }}</code>. Some documentation pages may
-        still state outdated information. This banner will tell you if you're reading one of such pages.
-      </p>
+        The content of this page was not yet updated for Godot
+        <code class="docutils literal notranslate">{{ godot_version }}</code>
+        and may be <strong>outdated</strong>. If you know how to improve this page or you can confirm
+        that it's up to date, feel free to <a href="https://github.com/godotengine/godot-docs">open a pull request</a>.
+      {% else %}
+      <p class="first admonition-title">Up to date</p>
       <p>
-        {% if meta and meta.get('article_outdated') == 'True' %}
-          The contents of this page can be <strong>outdated</strong>. If you know how to improve this page or you can confirm
-          that it's up to date, feel free to <a href="https://github.com/godotengine/godot-docs">open a pull request</a>.
-        {% else %}
-          The contents of this page are <strong>up to date</strong>. If you can still find outdated information, please
-          <a href="https://github.com/godotengine/godot-docs">open an issue</a>.
-        {% endif %}
+        This page is <strong>up to date</strong> for Godot <code class="docutils literal notranslate">{{ godot_version }}</code>.
+        If you still find outdated information, please <a href="https://github.com/godotengine/godot-docs">open an issue</a>.
+      {% endif %}
       </p>
```

Comment:
I'd move that little p inside of both blocks, so each version has complete and valid layout.